### PR TITLE
Added Straight hand and Straight flush functions

### PR DESCRIPTION
**Pull request by vondojr**

### CODE REVIEW

**Review by andrewskerratt — APPROVED** [review]
Nice code Joe and nice comments haha makes it easy to see what's going on. Defs a couple dif ways to find a straight but i like this method, well done!

File: source/Straight_hand.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
+from CardGames import *
+from Find_Flush import *
+
+
+def straight(player, river):
```
Comment (TMUProfLee) [inline source/Straight_hand.py:5]: There are some more efficient ways of doing this, but this is good and it works.

File: test/test_Straight_hand.html [hunk 2]
```diff
@@ -0,0 +1,282 @@
+<!DOCTYPE html>
```
Comment (JHerren726) [inline test/test_Straight_hand.html:1]: I like the test format.

**Review by JHerren726 — APPROVED** [review]
Looks great to me!

**Pull request merged**